go/lang/security/audit/crypto/use_of_weak_crypto_.yaml

```diff
@@ -0,0 +1,26 @@
+rules:
+- id: use-of-rc4
+  message: >-
+    Detected RC4 cipher algorithm which is insecure. The algorithm has many
+    known vulnerabilities. Use AES instead.
+  languages: [go]
+  severity: WARNING
+  metadata:
+    owasp:
+    - A03:2017 - Sensitive Data Exposure
+    - A02:2021 - Cryptographic Failures
+    cwe:
+    - 'CWE-327: Use of a Broken or Risky Cryptographic Algorithm'
+    source-rule-url: https://github.com/securego/gosec#available-rules
+    category: security
```

Comment:
also I think this rule is a copy of the one above 😓

Reply:
i created the new rules because there were missing checks so i create new rules.. :)

Reply:
welp!